blazor/diagram/connectors/segments/bezier/interaction/interaction.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Segments interaction in Blazor Diagram Component | Syncfusion
+description: Checkout and learn here all about Bezier Segments in Syncfusion Blazor Diagram component and much more details.
```

Comment:
description: Check out and learn about Bezier Segments in Syncfusion Blazor Diagram component and much more details.